dev.linwood.butterfly.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Ideally, you should maintain this file in your own repo and include it in the tarball.